notebook/static/notebook/js/searchandreplace.js

```diff
@@ -147,58 +147,54 @@ define(function(require){
    * Search N' Replace action handler.
    **/
   var snr = function(env, event) {
-    var search  = $("<input/>")
-      .addClass('form-control')
-      .attr('placeholder','Find');
+      
     var isRegExpButton = $('<button/>')
       .attr('type', 'button')
       .attr('id', 'isreg')
-      .addClass("btn btn-default")
+      .addClass("btn btn-default btn-sm")
       .attr('data-toggle','button')
+      .css('font-weight', 'bold')
       .attr('title', 'Use regex (JavaScript regex syntax)')
       .text('.*');
 
-    var onlySelectedButton = $('<button/>')
+    var onlySelectedButton = $('<button><i class="fa fa-align-left"/></button>')
       .attr('type', 'button')
-      .addClass("btn btn-default")
-      .append($('<i/>')
-        .addClass("fa fa-check-square-o")
-      )
+      .addClass("btn btn-default btn-sm")
       .attr('data-toggle','button')
-      .attr('title', 'Replace only in selected cells');
+      .attr('title', 'Find in selected cells');
```

Comment:
I would prefer replace, Find and replace does not have the "Find" functionality, if we put Find only people will expect using this dialog to jump to the position where the match is found.